doc/installation.rst

```diff
@@ -6,40 +6,39 @@ Installation
    To install RISE in development mode, see
    the `Developer Documentation <dev/index.rst>`_.
 
-From the most simple to the most complex one, you have 3 options:
+You essentially have 2 options:
 
-1 - Using conda (recommended)::
+1 - Using conda ::
 
- conda install -c damianavila82 rise
+ conda install -c conda-forge rise
 
-2 - Using pip (less recommended)::
-
- pip install RISE
-
-and then one more step to install the JS and CSS in the proper places::
+.. note::
 
- jupyter-nbextension install rise --py --sys-prefix
+   Previously, we provided packages in the `damianavila82` channel,
+   going forward please use the `conda-forge` channel because newest versions
+   will not be published in the `damianavila82` channel anymore.
 
-3 - The old way (are sure sure you want to go this path?):
+.. note::
 
-To install this nbextension, simply run ``python setup.py install`` from the
-*RISE* repository (please use the latest tag available or try master if you want).
+    If you are a Julia user, you can also install it from the Julia REPL with:
 
-And then the same two step described in the pip-based installation::
+    ```
+    using Conda
+    Conda.add_channel("conda-forge")
+    Conda.add("rise")
+    ```
 
- jupyter-nbextension install rise --py --sys-prefix
+2 - Using pip ::
 
-and::
+ pip install RISE
 
- jupyter-nbextension enable rise --py --sys-prefix
+.. note::
 
-Conclusion: If you use conda, life will be easy ;-)
+   Before RISE 5.4.2, it was necessary to perform one more step to install the JS and CSS in the proper places with ::
 
-.. note::
+     jupyter-nbextension install rise --py --sys-prefix
 
-   In all the options available the ``--sys-prefix`` option will install and
-   enable the extension in the current environment, if you want a ``--user`` based or a
-   ``--system`` based installation just use those option instead in the above commands.
+   This is **no** needed anymore because those resources are installed automatically by the setup.py.
```

Comment:
there's a typo here:
you mean either **no longer** needed, or **not needed** anymore

Reply:
Already fixed on #465, but thanks for the ping!